src/app/component/Field/Field.component.js

```diff
@@ -332,7 +365,14 @@ Field.propTypes = {
         PropTypes.func,
         PropTypes.shape({ current: PropTypes.instanceOf(Element) })
     ]),
-    isAutocompleteAllowed: PropTypes.bool
+    isAutocompleteAllowed: PropTypes.bool,
+    countries: PropTypes.arrayOf(
+        PropTypes.shape({
+            id: PropTypes.string,
+            label: PropTypes.string
+        })
+    ).isRequired,
+    getCountriesList: PropTypes.func.isRequired
```

Comment:
Why this one is here? This logic should be encapsulated somewhere, not in abstract field component.